mlflow/shap.py

```diff
@@ -239,6 +240,12 @@ def predict_function(X) -> pred:
     import matplotlib.pyplot as plt
     import shap
 
+    original_config_dict = mlflow.utils.autologging_utils.AUTOLOGGING_INTEGRATIONS
+
+    new_config = original_config_dict.copy()
+    for integration in original_config_dict:
+        new_config[integration]["disable"] = True
+
```

Comment:
@mohamad-arabi I'm actually thinking it might be better to put this logic behind an `autologging.disable()` function so that we can use it like:
```
with autologging.disabled():
	...
```
because I suspect we'll need to do this in other places. Also it'd be nicer to keep the `AUTOLOGGING_INTEGRATIONS` abstracted away where it is.